tests/unit/src/main/scala/tests/BaseLspSuite.scala

```diff
@@ -48,6 +49,9 @@ abstract class BaseLspSuite(
 
   protected def initializationOptions: Option[InitializationOptions] = None
 
+  protected def mtagsResolver: MtagsResolver =
+    MtagsResolver.forUnitTests()
```

Comment:
We could have the actual implementation kept here.

Reply:
Initially I defined it here but then I realized that it's needed to be used in ProblemSuite too. 
So I decided that it would be better to move it into MtagsResolver . Just to make it more clear that there is such instance.

Reply:
Sure, though I would keep it in tests if possible

Reply:
Ok, moved to tests